### PR TITLE
Add nation and HHS to Safegraph

### PR DESCRIPTION
**Pull request by chinandrew**
### Description
**Merge after #610**

Add HHS and nation to safegraph. There's some test code duplication that can be removed that I didn't bother with in this PR.

### Changelog
Itemize code/test/documentation changes and files added/removed.
- Update conditional statements to map to nation and hhs
- Update tests to run the hhs and nation cases

### Fixes 
- n/a

### CODE REVIEW

File: safegraph/DETAILS.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 We import census block group-level raw mobility indicators from Safegraph,
 calculate functions of the raw data, and then aggregate the data to the
-county and state levels.  MSA and HRR not yet implemented.
+county and state levels.
```
Comment (jingjtang) [inline safegraph/DETAILS.md:5]: Better change to "aggregate the data to the county, MSA, HRR, state, HHS and national levels."

**Comment by jingjtang** [comment]
tests passed. 
linting looks good.

File: safegraph/DETAILS.md (hunk 1 above)
Comment (chinandrew) [reply safegraph/DETAILS.md]: thanks, updated

**Review by jingjtang — APPROVED** [review]
lgtm

**Pull request merged**